src/imitation/scripts/train_rl.py

```diff
@@ -53,7 +54,9 @@ def train_rl(
 
     Args:
         total_timesteps: Number of training timesteps in `model.learn()`.
-        normalize_reward: If True, then rescale and clip reward.
+        normalize_reward: Applies normalization and clipping to the reward function by
```

Comment:
Good point! Thanks for making a note about the difference between `VecNormalize` and `RunningNorm`. Though the difference is minor, I would suggest making an issue to track this difference and considering wrapping the `VecEnv` using a `RewardVecEnvWrapper` that takes in a `reward_fn` that takes in the ground truth reward and pass through a normalization layer.

This would be also useful if we want to use EMWA as the normalization layer for `train_rl`.

Reply:
We could do that for consistency, but it seems a bit beyond the scope of what we are doing in this pull request. I've opened another issue to discuss it #442 like you suggested.